solutions/cses-1084.mdx

```diff
@@ -2,44 +2,63 @@
 id: cses-1084
 source: CSES
 title: Apartments
-author: Danh Ta Chi Thanh
+author: Nathan Gong, Danh Ta Chi Thanh
 ---
 
-## Solution
-
 [Unofficial Editorial](https://codeforces.com/blog/entry/83295)
 
+We can use a greedy approach to solve this problem. First, sort the applicant
+and apartment arrays. We can keep two pointers $i$ and $j$ (initialized to 0),
+which keep track of the current index of the applicant and apartment we are
+looking at respectively. Then, while there are more applicants and apartments to
+look through, we repeatedly run the following check:
+
+- If $|\texttt{applicants}[i] - \texttt{apartments}[j]| <= k$, we have found a
+  suitable apartment for the current applicant. Thus, we increment $i$, $j$, and
+  our answer.
+- Otherwise, $\texttt{apartments}[j]$ is either too big or too small for
+  $\texttt{applicants}[i]$. We can increment $i$ and $j$ accordingly.
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(n\log(n)+m\log(m))$
+
+<LanguageSection>
+
+<CPPSection>
+
 ```cpp
 #include <bits/stdc++.h>
 
 using namespace std;
 
-//constant initialization
-const int maxn=2e5+10;
+// Constant initialization
+const int maxn = 2e5 + 10;
```

Comment:
```suggestion
const int MAX_N = 2e5
```
think this should still work? the +10 makes readers question "wait why is the +10 there"
it's like adding unnecessary supports, people will assume that w/o them the building would fail and then look to see why they're necessary